__tests__/app.test.js

```diff
@@ -392,6 +392,41 @@ describe('GET /api/users', () =>{
         })
     });
 })
+//11
+describe('GET /api/articles query topic', () => {
+    test('GET:200 sends an appropriate status and article array on the certain topic', () => {
+        return request(app)
+        .get('/api/articles?topic=cats')
+        .expect(200)
+        .then(({body}) => {         
+            expect(body.articles).toHaveLength(1)
+
+            body.articles.forEach((article) => {
+                expect(article.topic).toEqual('cats')
+            })
+        })
+    });
+    test('GET:200 sends an appropriate status and article array on the certain topic', () => {
+        return request(app)
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+        .then(({body}) => {         
+            expect(body.articles).toHaveLength(12)
+            body.articles.forEach((article) => {
+                expect(article.topic).toEqual('mitch')
+            })
+        })
+    });
+    test('GET 200 if topic does not exist sends array of all articles',  () => {
```

Comment:
I would send back a 404 to the client, as seeing all the articles wouldn't be very clear 👍

Reply:
Nice work, don't forget to change the description on line 420